adminSiteClient/DimensionCard.tsx

```diff
@@ -249,6 +249,7 @@ export class DimensionCard<
                             store={dimension.display}
                             auto={column.tolerance}
                             onBlur={this.onChange}
+                            helpText="May be overridden by the map's tolerance setting"
```

Comment:
Maybe it's better to clarify: "Tolerance for charts (bar, marimekko, [others?]) and maps. The tolerance set in the Map tab, if defined, overrides the map tolerance defined here." (feel free to rephrase or shorten if you think this is too much detail).